code/modules/mob/living/silicon/laws.dm

```diff
@@ -90,6 +91,13 @@
 	post_lawchange(announce)
 
 /mob/living/silicon/proc/make_laws()
+	if(SSticker.current_state < GAME_STATE_PLAYING)
+		if(!laws)
+			addtimer(CALLBACK(src, /mob/living/silicon/.proc/make_laws), 3 SECONDS) // game data should be established before game started, then the data can be used to law setup
+	else
+		make_laws_force()
```

Comment:
because laws are created before roundstart, while the racimov laws need game data after roundstart, I had to make `make_laws()` delayed so that it can find data.
There was nothing particularly going bad at a test.

<details>
<summary>See the late law establish result</summary>

![image](https://user-images.githubusercontent.com/87972842/175858882-4ed6ae35-3514-4233-9a71-2c430042b67a.png)

There wasn't that much gap time to have no laws at the moment.

</details>

I know this isn't that elegant, but refactoring the spawning system is too difficult, and even can be more expensive than this.